query/src/main/scala/filodb/query/PromQueryResponse.scala

```diff
@@ -4,18 +4,23 @@ sealed trait PromQueryResponse {
   def status: String
 }
 
-final case class RemoteErrorResponse(status: String, errorType: String, error: String) extends PromQueryResponse
+final case class RemoteErrorResponse(status: String, errorType: String, error: String,
+                                     queryStats: Option[Seq[RemoteQueryStats]] = None) extends PromQueryResponse
 
 final case class ErrorResponse(errorType: String, error: String, status: String = "error") extends PromQueryResponse
 
 final case class SuccessResponse(data: Data, status: String = "success",
                                  partial: Option[Boolean] = None,
-                                 message: Option[String] = None) extends PromQueryResponse
+                                 message: Option[String] = None,
+                                 queryStats: Option[Seq[RemoteQueryStats]] = None) extends PromQueryResponse
 
 final case class ExplainPlanResponse(debugInfo: Seq[String], status: String = "success",
                                      partial: Option[Boolean]= None,
                                      message: Option[String]= None) extends PromQueryResponse
 
+final case class RemoteQueryStats(group: Seq[String], timeSeriesScanned: Long,
```

Comment:
Rename to `QueryStatistics`. If possible, we can try to use the existing `QueryStats` object. This will avoid translation

Reply:
Renamed to QueryStatistics. We can not reuse `QueryStats` since it uses `TrieMap` which is difficult to convert to/from json object.